docs/docs/overview/does-testkube-replace-cicd.md

```diff
@@ -0,0 +1,9 @@
+# Does Testkube replace my CI/CD?
+
+The beauty of Testkube is that it improves your CI/CD pipeline!
+
+There's different ways to integrate Testkube with your current CI/CD. You can directly use the CLI, or if you use GitHub, you can create GitHub Actions. Read more about the process [here](https://kubeshop.github.io/testkube/integrations/testkube-automation).
+
+If you're familiar with GitOps, check the [ArgoCD integration guide](https://testkube.kubeshop.io/blog/a-gitops-powered-kubernetes-testing-machine-with-argocd-and-testkube) will be useful, and check or the [Flux integration guide](https://testkube.io/blog/flux-testkube-gitops-testing-is-here). The diagram of GitOps + Testkube would look like this: 
```

Comment:
If you're familiar with GitOps, [ArgoCD integration guide](https://testkube.kubeshop.io/blog/a-gitops-powered-kubernetes-testing-machine-with-argocd-and-testkube) will be useful, and check the [Flux integration guide](https://testkube.io/blog/flux-testkube-gitops-testing-is-here). The diagram of GitOps + Testkube would look like this: